ETL/firebase_migration_v2.py

```
@@ -1672,15 +1716,27 @@ def _clean_name(value: Any) -> str:
+				end_date = raw_end.date().strftime("%m/%d/%Y")
+			elif isinstance(raw_end, date):
+				end_date = raw_end.strftime("%m/%d/%Y")
+			else:
+				text = str(raw_end).strip()
+				parsed_end = None
+				for fmt in ("%m/%d/%Y", "%Y-%m-%d", "%Y-%m-%d %H:%M:%S"):
+					try:
+						parsed_end = datetime.strptime(text, fmt).date()
+						break
+					except Exception:
+						continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)